lib/beaker-hostgenerator/data.rb

```diff
@@ -829,62 +588,6 @@ def osinfo
             'template' => 'oracle-7-x86_64',
           },
         },
-        'osx109-64' => {
-          :general => {
-            'platform' => 'osx-10.9-x86_64',
-          },
-          :vmpooler => {
-            'template' => 'osx-109-x86_64',
-          },
-        },
-        'osx1010-64' => {
-          :general => {
-            'platform' => 'osx-10.10-x86_64',
-          },
-          :vmpooler => {
-            'template' => 'osx-1010-x86_64',
-          },
-        },
-        'osx1011-64' => {
-          :general => {
-            'platform' => 'osx-10.11-x86_64',
-          },
-          :vmpooler => {
-            'template' => 'osx-1011-x86_64',
-          },
-        },
-        'osx1012-64' => {
-          :general => {
-            'platform' => 'osx-10.12-x86_64',
-          },
-          :vmpooler => {
-            'template' => 'osx-1012-x86_64',
-          },
-        },
-        'osx1013-64' => {
-          :general => {
-            'platform' => 'osx-10.13-x86_64',
-          },
-          :vmpooler => {
-            'template' => 'osx-1013-x86_64',
-          },
-        },
-        'osx1014-64' => {
-          :general => {
-            'platform' => 'osx-10.14-x86_64',
-          },
-          :vmpooler => {
-            'template' => 'osx-1014-x86_64',
-          },
-        },
-        'osx1015-64' => {
-          :general => {
-            'platform' => 'osx-10.15-x86_64',
-          },
-          :vmpooler => {
-            'template' => 'osx-1015-x86_64',
-          },
-        },
```

Comment:
While [we've announced end of support for Catalina](https://groups.google.com/g/puppet-announce/c/Un-JAtMEvAc), we haven't quite removed it from our pipelines. Would you mind holding off on this for a little while longer?

Reply:
So is that just 10.15 or all versions?

Reply:
Sorry, to be clear my preference would be to leave macOS 10.15 and up (10.15 Catalina, 11 Big Sur, 12 Monterey) as-is.

Reply:
Removing 10.15 is blocked on https://tickets.puppetlabs.com/browse/PA-5024. All of the prior versions can be deleted.